app/frontend/src/search/hits.js

```diff
@@ -0,0 +1,48 @@
+export const renderHits = (renderOptions) => {
+  const { results, widgetParams } = renderOptions;
+
+  if (results) {
+    widgetParams.container.innerHTML = `There are ${results.nbHits} jobs listed`
+  }
+};
+
+export const formatSnakeCase = value => value.toLowerCase().replace(/_/g, ' ')
```

Comment:
I found the name `formatSnakeCase` a little bit confusing.  It might be clearer if it were `reFormatSnakeCase` or maybe `snakeCaseToHumanReadable`.